app/components/work_history_item_component.html.erb

```diff
@@ -10,7 +10,14 @@
   </p>
   <p class="govuk-body">
     <%= details %>
-    <% if working_with_children? %>
+    <% if FeatureFlag.active?(:restructured_work_history) && relevant_skills? %>
```

Comment:
Do we need the feature flag here? Once we turn it on we'll still have to maintain backwards compatibility

Reply:
:+1: No I don't think it does, these conditional methods are guarded enough to fall through to the old flag.